app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(8, 1)) == 8
```

Comment:
I think 1 as the second parameter in this test is a bad idea. What do you think?